src/utilities/utilities.js

```diff
@@ -197,4 +217,68 @@ export const collectionlLinks = [
     ),
     tooltipText: 'View Collection in Cyberconnect.me'
   },
+  {
+    id: 'GM',
+    getUrl: GmLink,
+    icon: (
+      <Gm
+        style={{
+          margin: "0 5px",
+          verticalAlign: "middle",
+          width: "20px",
+          height: "20px",
+        }}
+        alt={"Open external link"}
+      />
+    ),
+    tooltipText: 'View Collection in Gm.xyz'
+  },
+  {
+    id: 'Moca',
+    getUrl: MocaLink,
+    icon: (
+      <Moca
+        style={{
+          margin: "0 5px",
+          verticalAlign: "middle",
+          width: "20px",
+          height: "20px",
+        }}
+        alt={"Open external link"}
+      />
+    ),
+    tooltipText: 'View Collection in Museumofcryptoart.com'
+  },
+  {
+    id: 'Welook',
+    getUrl: WelookLink,
+    icon: (
+      <Welook
+        style={{
+          margin: "0 5px",
+          verticalAlign: "middle",
+          width: "20px",
```

Comment:
In this case, the Welook logo is wider than the rest. Try using a higher width to avoid breaking the aspect ratio.

Reply:
Done